addons/point_of_sale/static/src/js/models.js

```diff
@@ -1557,7 +1723,7 @@ exports.Product = Backbone.Model.extend({
     // ORM. After that they are added in this order to the pricelists.
     get_price: function(pricelist, quantity, price_extra){
         var self = this;
-        var date = moment();
+        var date = moment().startOf('day');
```

Comment:
I don't know if this has been asked, but are you sure about this? It seems that the original method `get_product_price` - one from `product.pricelist` - doesn't use "start of day", it used "fields.Datetime.now()" as default.